infra/bridge/src/controllers/hooks/telegram.js

```diff
@@ -83,51 +111,39 @@ router.post('/', (req, res) => {
   if (isValidGroup && message.new_chat_members) {
     // For join verifications
     const events = message.new_chat_members
+    const chunks = _chunk(
```

Comment:
Super nit: explain why you are chunking the writes (to avoid creating too many DB connections in case we receive a very large number of events at once).